spaceship/views.py

```diff
@@ -68,17 +68,36 @@ def home():
 
 @app.route('/login', methods=['GET', 'POST'])
 def login():
+  oauth_user_info = session.pop('oauth_user_info', None)
```

Comment:
was confused about this line going here, because i thought this has some effect on the next line (`current_user.is_authenticated`). i would put it lower down, where you initialize `email_address`

Reply:
done